types/mjml-core/index.d.ts

```diff
@@ -0,0 +1,254 @@
+// Type definitions for mjml-core 4.7
+// Project: https://mjml.io
+// Definitions by: Ian Edington       <https://github.com/IanEdington>
+//                 aahoughton         <https://github.com/aahoughton>
+//                 marpstar           <https://github.com/marpstar>
+//                 eiskalteschatten   <https://github.com/eiskalteschatten>
+//                 emrah88            <https://github.com/emrah88>
```

Comment:
@aahoughton @marpstar @eiskalteschatten @emrah88 I've added you as maintainers on mjml-core since you contributed to mjml and it's basically the same package, with a few additions.